packages/apputils/test/sanitizer.spec.ts

```diff
@@ -6,6 +6,20 @@ const sanitizer = new Sanitizer();
 
 describe('sanitizer', () => {
   describe('#sanitize()', () => {
+    it('should preserve links containing user-defied schemes when named properties are allowed', () => {
+      const customSanitizer = new Sanitizer();
+      customSanitizer.setAllowedSchemes([
+        'http', 'https', 'mailto', 'ftp', 'tel', 'zoommtg'
+      ]);
+      customSanitizer.setAllowNamedProperties(true);
+
+      const link = '<a href="zoommtg://meeting_link">Join Meeting</a>';
+      const result = customSanitizer.sanitize(link);
+
+      expect(result).toBe(link);
```

Comment:
Looks like the test is expecting an extra `nofollow`:

![image](https://github.com/user-attachments/assets/fb89b2dc-b14f-4f22-bd17-9205f40792e2)